module/mailer/src/Resources/config/messenger.yaml

```diff
@@ -0,0 +1,4 @@
+framework:
+    messenger:
+        routing:
+            Ergonode\Mailer\Domain\Command\SendMailCommand: '%ergonode.mailer.messenger_transport_name%'
```

Comment:
There are no `transports` to mail.

Reply:
there's `notification` and that's the one under parameter